src/rpc/mining.cpp

```diff
@@ -94,10 +94,11 @@ static UniValue getnetworkhashps(const JSONRPCRequest& request)
                 {
                     {"nblocks", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "120", "The number of blocks, or -1 for blocks since last difficulty change."},
                     {"height", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "-1", "To estimate at the time of the given height."},
-                }}
-                .ToString() +
-            "\nResult:\n"
+                },
+                RPCResult{
```

Comment:
Drop `RPCResult`?

Reply:
I think it is better to label the sections instead of having a jungle of `{}`

Reply:
It's pretty obviously what always comes after args but ok.